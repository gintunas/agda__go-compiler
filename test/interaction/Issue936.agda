module Issue936 where

-- The highlighting generated for this file should not contain
-- any references to Agda.Primitive.
