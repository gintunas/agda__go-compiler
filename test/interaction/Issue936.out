(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-highlight-add-annotations 'nil '(1 7 (keyword) t) '(17 22 (keyword) t) '(24 86 (comment) t) '(87 123 (comment) t))
(agda2-highlight-add-annotations 'nil '(1 7 (keyword) t) '(8 16 (module) nil nil ("Issue936.agda" . 1)) '(17 22 (keyword) t))
(agda2-highlight-add-annotations 'nil '(1 7 (keyword) t) '(8 16 (module) nil nil ("Issue936.agda" . 1)) '(17 22 (keyword) t))
(agda2-highlight-add-annotations 'nil '(24 86 (comment) t) '(87 123 (comment) t))
(agda2-status-action "Checked")
(agda2-info-action "*All Done*" "" nil)
((last . 1) . (agda2-goals-action '()))
